src/components/Search/Search.js

```diff
@@ -1,17 +1,40 @@
-import React from 'react'
-import './Search.css'
+import React, { useState, useEffect } from "react";
+import "./Search.css";
+
+const Search = ({ setDrinks, drinks }) => {
+  const [searchInput, setSearchInput] = useState("");
+  const [error, setError] = useState("");
+
+  const handleSearch = e => {
+    e.preventDefault();
+    {
```

Comment:
You could consider adding a function that that updates the search made where your drinks state lives to dynamically update the results.